src/sentry/tasks/post_process.py

```diff
@@ -945,13 +945,15 @@ def _get_replay_id(event):
 
 
 def process_workflow_engine(job: PostProcessJob) -> None:
-    if job["is_reprocessed"]:
-        return
+    """
```

Comment:
This was just DRYing up the code in post process, eventually we could use this method for all hooks into the detector system here.